src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt

```diff
@@ -0,0 +1,51 @@
+package org.kamiblue.botkt.command.commands
+
+import org.kamiblue.botkt.Main
+import org.kamiblue.botkt.command.BotCommand
+import org.kamiblue.botkt.utils.Colors
+
+object HelpCommand : BotCommand(
+    name = "help-mod",
```

Comment:
default name should be "commands", and add an alias for "help" and "help-mod"

Reply:
“help" is gonna be conflicting with the js bot. And it is HelpCommand not "CommandsCommand"

Reply:
I know it'll conflict, I'll rename the bot-js one

Reply:
bot-kt is a replacement for bot-js

Reply:
🗿